operator-common/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/CruiseControlConfigurationGoals.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.operator.cluster.operator.resource.cruisecontrol;
+
+public enum CruiseControlConfigurationGoals {
```

Comment:
I would just call this `CruiseControlGoals`

Reply:
fixed